content/en/armory-enterprise/armory-admin/policy-engine/policy-engine-enable/policy-engine-plug-enable.md

```diff
@@ -296,23 +134,7 @@ You can configure the amount of time that the Policy Engine waits for a response
 {{% /tabbody %}}
 {{< /tabs >}}
 
-### Remote plugin repository
-
-The configuration is mostly identical to the Docker image method but omits all volumes and init container configurations. Additionally, replace all occurrences of the following:
-
```

Comment:
I think we can do something LIKE:

```
### Local plugin installation
It is possible to install the plugin by using init containers and custom volume mounts.  This is a more complex installation method.  If it's needed compliance or other reasons, please look at KB articles on patching the init containers, volumes and resources for spinnaker... 
```
This might be useful in general on some of our docs?  Thoughts?